deploy/ansible/hosts/feature.yml

```diff
@@ -35,7 +35,7 @@ goshimmers:
 
   vars:
     analysisServerHost: "analysisentry-01.feature.shimmer.iota.cafe"
-    entryNodeHost: "analysisentry-01.feature.shimmer.iota.cafe"
+    entryNodeHost: "bootstrap-01.feature.shimmer.iota.cafe"
```

Comment:
Why is it so now?

Reply:
It's not like that. I was just testing to see why it doesn't do initial peering.